### PR TITLE
Other functionality amends

### PR DESCRIPTION
**Pull request by paulhowes-HMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4636

### Change description ###
Booking location and change link functionality and additional styling.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/booking/containers/booking-date/booking-date.component.ts [hunk 1]
```diff
@@ -74,10 +74,24 @@ export class BookingDateComponent implements OnInit {
...
+      if (this.formGroup.get(DateFormControl.BOOKING_DATE_TYPE).value === BookingDateOption.DATERANGE && this.bookingProcess.startDate) {
+        this.displayBookingDates();
+      }
     }
     this.setValidators();
   }
 
+  private displayBookingDates(): void {
+    this.formGroup.get(DateFormControl.BOOKING_START_DAY).setValue(this.bookingProcess.startDate.getDate().toString().padStart(2, '0'));
+    this.formGroup.get(DateFormControl.BOOKING_START_MONTH).setValue((this.bookingProcess.startDate.getMonth() + 1).toString().padStart(2, '0'));
```
Comment (adnanakgun) [inline src/booking/containers/booking-date/booking-date.component.ts:88]: What are submitting at the very end? Date or string?

**Review by adnanakgun — APPROVED** [review]
Approved with comments

**Review by tarikdogan-hmcts — APPROVED** [review]

File: src/booking/containers/booking-date/booking-date.component.ts (hunk 1 above)
Comment (paulhowes-HMCTS) [reply src/booking/containers/booking-date/booking-date.component.ts:88]: Date - this merely splits the date in to three input strings.

File: src/booking/containers/booking-wrapper/booking-wrapper.component.ts [hunk 2]
```diff
@@ -51,7 +51,7 @@ export class BookingWrapperComponent implements OnInit {
             throw new Error('Invalid Booking Back state');
         }
         break;
-      case BookingNavigationEvent.HOMECONTINUE:
+      case BookingNavigationEvent.HOMECONTINUE: case BookingNavigationEvent.CHANGELOCATIONCLICK:
```
[the remark this answers is not in the document]
Comment (paulhowes-HMCTS) [reply src/booking/containers/booking-wrapper/booking-wrapper.component.ts:54]: I've moved it onto a new line to make it more readable

**Pull request merged**